README.md

```diff
@@ -11,6 +11,8 @@ You can see a live example of it working on any of this sites (there are many ju
 ## Usage
 
 ```js
+import xlsx from "json-as-xlsx-wm"
```

Comment:
"json-as-xlsx-wm" is your package? 🤔

Reply:
Yes, this is my package. The usual inattention. Need to fix the package name